cluster/sync-common.sh

```diff
@@ -0,0 +1,94 @@
+function eventually {
+    timeout=$(( $KUBEVIRT_NUM_NODES * 10 ))
+    interval=5
+    cmd=$@
+    echo "Checking eventually $cmd"
+    while ! $cmd; do
+        sleep $interval
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 1
+        fi
+    done
+}
+
+function consistently {
+    timeout=15
```

Comment:
and maybe we can use for loop here instead of the while?

Reply:
Well we are testing `cmd` so feels better have it at the loop condition.